views/components/certification/Cert.js

```diff
@@ -0,0 +1,58 @@
+import React, { Fragment } from 'react'
+import styled from 'styled-components/macro'
+import CertIcon from '~/views/components/certification/CertIcon'
+import Info from '~/views/components/certification/Info'
+import Title from '~/views/components/certification/Title'
+import Item from '~/views/components/certification/Item'
+import { CERT_ACTIVE_ITEM } from '~/consts/certification'
+import { leftInfo } from '~/consts/certification'
+import { zIndexArr } from '~/consts/certification'
+import { leftPos } from '~/consts/certification'
+import PropTypes from 'prop-types'
+import { Translate } from 'react-localize-redux'
+
+const Cert = ({ itemSelected, setSelectedItem, title, content, id, name }) => {
+  return (
+    <Fragment>
+      <CertItem id={id} onClick={() => setSelectedItem(id)}>
+        <CertIcon id={id} itemSelected={itemSelected} name={name} />
+      </CertItem>
+
+      <CertInfo itemSelected={itemSelected} id={id}>
+        <Title>
+          <span>{id}</span> &mdash; <Translate id={title} />
+        </Title>
+        <p>
+          <Translate id={content} />
+        </p>
+      </CertInfo>
+    </Fragment>
+  )
+}
+
+export default Cert
+
+Cert.propTypes = {
+  itemSelected: PropTypes.any,
```

Comment:
any is not good here, we want specific type if value go in

Reply:
Update to PropTypes.number